driver-sync/build.gradle

```diff
@@ -43,9 +43,12 @@ tasks.withType(Checkstyle) {
 
 jar {
     bnd (
-        '-exportcontents': "*;-noimport:=true",
+        '-exportcontents': "*",
```

Comment:
Why do some modules still contain `noimport=true`, and some do not?  What's the effect on the manifest for each module?